metagpt/tools/__init__.py

```diff
@@ -19,6 +19,7 @@ class SearchEngineType(Enum):
     DIRECT_GOOGLE = "google"
     DUCK_DUCK_GO = "ddg"
     CUSTOM_ENGINE = "custom"
+    Bing = "bing"
```

Comment:
It is recommended to use all uppercase for enum names. Update to `BING = "bing"`.

Reply:
> It is recommended to use all uppercase for enum names. Update to `BING = "bing"`.

Thank you very much for the reminder🙏